tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/crop_view.py

```diff
@@ -28,11 +29,13 @@ def __init__(self, marionette):
         Wait(self.marionette).until(expected.element_enabled(done))
 
     def tap_crop_done(self):
-        try:
-            self.marionette.find_element(*self._crop_done_button_locator).tap()
-        except (FrameSendFailureError, NoSuchWindowException):
-            # The frame may close for Marionette but that's expected so we can continue - Bug 1065933
-            pass
+        # Workaround for bug 1161088, where tapping on the button inside the app itself
+        # makes Marionette spew out NS_ERROR_NOT_INITIALIZED errors
+        element = self.marionette.find_element(*self._crop_done_button_locator)
+        x = element.rect['x'] + 5
+        y = element.rect['y'] + 5
```

Comment:
Yes, I can do that. I have to admit, it was just laziness, adding the 5 pixels instead of trying to click the center of the element.